bayesian/api/api_v2.py

```diff
@@ -247,10 +219,11 @@ def post():
         # Step4: Handle Unknown Packages
         if unknown_pkgs:
             stack_recommendation = add_unknown_pkg_info(stack_recommendation, unknown_pkgs)
-            if os.environ.get("DISABLE_UNKNOWN_PACKAGE_FLOW") != "1":
-                # Unknown Packages is Present and INGESTION is Enabled
-                logger.debug(unknown_pkgs)
-                unknown_package_flow(ecosystem, unknown_pkgs)
+            pkgs_to_ingest = set(map(lambda pkg: ingestion_utils.Package(package=pkg.package,
+                                                                         version=pkg.version),
+                                     unknown_pkgs))
+            logger.debug(pkgs_to_ingest)
```

Comment:
Add some info on what you are logging.
```suggestion
            logger.debug("Unknown ingestion triggered for %s", pkgs_to_ingest)
```